src/chop/nn/snn/auto_cuda/base.py

```diff
@@ -0,0 +1,1548 @@
+import numpy as np
```

Comment:
Is this file copied from some package (`spikingjelly`)? If so, we may leave some references in this file and `README.md` for further maintenance & copyright stuff.

Reply:
Yes, add some code headers and a readme.